ChangeLog.d/configuration-split.txt

```diff
@@ -0,0 +1,8 @@
+Changes
+   * Configuration entries have been migrated from MBedTLS library
+     configuration file (include/mbedtls/mbedtls_config.h)
+     to the tf-psa-crypto one (tf-psa-crypto/include/psa/crypto_config.h)
+     as per design proposal (PR #9236). The test suites, as the built-in
+     and external (TF-M) configurations have been updated accordingly.
+     Configurations using `MBEDTLS_PSA_CRYPTO_CONFIG_FILE` as a pre-processor
+     directive, will have to specify the file's location using the build-system.
```

Comment:
The last sentence seems too specific to me. I'd go for something more general like:
```suggestion
     To migrate custom Mbed TLS configurations where
     MBEDTLS_PSA_CRYPTO_CONFIG is disabled, you should first adapt them
     to the PSA configuration scheme based on PSA_WANT_XXX symbols
     (see psa-conditional-inclusion-c.md for more information).
     To migrate custom Mbed TLS configurations where
     MBEDTLS_PSA_CRYPTO_CONFIG is enabled, you should migrate the
     cryptographic and platform configuration options from mbedtls_config.h
     to crypto_config.h (see config-split.md for more information and configs/
     for examples).
```